module_utils/helpers.py

```diff
@@ -415,6 +429,22 @@ def get_cluster_replicasets(control_console):
     return cluster_replicasets
 
 
+def set_twophase_options(control_console, options):
+    control_console.eval_res_err(SET_TWOPHASE_OPTIONS_FUNC_BODY, options)
+
+
+def set_twophase_options_from_params(control_console, params):
+    twophase_options = {}
+    if params.get('netbox_call_timeout') is not None:
+        twophase_options['netbox_call_timeout'] = params['netbox_call_timeout']
+    if params.get('upload_config_timeout') is not None:
+        twophase_options['upload_config_timeout'] = params['upload_config_timeout']
+    if params.get('apply_config_timeout') is not None:
+        twophase_options['apply_config_timeout'] = params['apply_config_timeout']
```

Comment:
```py
allowed_options = [...]
twophase_options = {}
for option in allowed_options:
    if params.get(option) is not None:
        twophase_options[option] = params[option]
```

Reply:
The point is that specifically these options cannot be None, so we check them for None to filter.  Other options may be nullable, so they should not be checked.  Perhaps it is better to write that this is not_null_options